### DISCUSSION TITLE
Install Neuron

### DISCUSSION THREAD

**Discussion by megurel** [srid/neuron#617] [Installation]
Hi,

Can someone help with the installation process? I followed the steps but could not. I have nothing idea for the programming, unfortunately, I just want to use neuron on my own page to take notes. Thank you in advance for your return.

i maked just this step : https://megurel.github.io/mynotes/

**Comment by srid** [comment]
Have you read the section "How to edit and add notes" under https://megurel.github.io/mynotes/README ?

**Reply by srid** [reply]
It doesn't explain it clearly, but the edit button is next to the home one, here:

![image](https://user-images.githubusercontent.com/3998/118300217-af676f00-b4af-11eb-9260-5b726c1ab28a.png)

**Reply by srid** [reply]
There is also https://www.cerveau.app/ - if you want a fully online editing experience.

**Reply by megurel** [reply]
When i clicked this ![image](https://user-images.githubusercontent.com/40196972/118358212-806ffc80-b586-11eb-81f5-0f6c916536eb.png)
 button. The following page is coming.

![image](https://user-images.githubusercontent.com/40196972/118358183-5ae2f300-b586-11eb-9ed9-2766bb48b5e5.png)

what should i do next? Thanks for your help. (address is this = https://github.com/megurel/mynotes/blob/master/index.md)

**Reply by srid** [reply]
@megurel Try this, https://github.com/megurel/mynotes/pull/1

You had "view" url. We need "edit" url for github.

**Reply by megurel** [reply]
Sorry to keep you busy. :) I agreed to edit and the address was like this: 

https://github.com/megurel/mynotes/edit/master/index.md

and the screen is here again, this time in the index.md of editing.


![image](https://user-images.githubusercontent.com/40196972/118373503-e88f0480-b5bf-11eb-8fe9-3b86da4ab96d.png)

**Reply by srid** [reply]
No worries; so looks like it is working for you. Just make your edits, and click the green "Commit changes" button (which I believe should be below the edit box, and not shown in your screenshot). The site will update in a minute or two.

**Reply by megurel** [reply]
Thanks a lot, now its work :) will i editing at github ? in your demo it felt that like it has its own editing part. And besides, what is the most appropriate and private way to use the neuron? Github paid for private. Thanks a lot for your help... I learned something thanks to you.

**Reply by srid** [reply]
If you are referring to the [demo here](https://notes.srid.ca/cerveau-announce), then that's from https://www.cerveau.app/ (you can sign-in with GitHub and follow the instructions to install it on your repository).

As for private notebooks, well - you can mark the GitHub repo as private - but the Pages site will require a Pro plan (and I believe it will be publicly accessible). However, you can use Cerveau on private repositories to edit/add notes. Failing that, your best bet is to install neuron on your computer, and use VSCode (or Obsidian) as notebook editor (which is what I do, mostly).
